content/docs/0.5.3-powered_by.html

```diff
@@ -462,13 +462,17 @@ <h2 id="talks--presentations">Talks &amp; Presentations</h2>
   <li>
     <p><a href="https://drive.google.com/open?id=1Pk_WdFxfEZxMMfAOn0R8-m3ALkcN6G9e">“Building a near real-time, high-performance data warehouse based on Apache Hudi and Apache Kylin”</a> - By ShaoFeng Shi March 2020, Apache Hudi &amp; Apache Kylin Online Meetup, China</p>
   </li>
+  <li>
+    <p><a href="https://drive.google.com/file/d/1K-WsQAQf-C96pLRTpM4Ha1v7wRf1pZ9S/view?usp=sharing">“Apache Hudi learning series: Understanding Hudi internals”</a> - By Abhishek Modi, Balajee Nagasubramaniam, Prashant Wason, Satish Kotha, Nishith Agarwal Feb 2021, Uber Meetup, Virtual</p>
```

Comment:
hi @n3nash, the right way is to add this to `docs\_docs\0.5.3\1_4_powered_by.md`

Reply:
Thanks @lamber-ken for pointing that out. I've made those changes, can you take a look ?

Reply:
👍